frontend/src/forms/LeadForm.jsx

```diff
@@ -44,19 +46,20 @@ export default function LeadForm() {
 
       <Form.Item
         label={translate('phone')}
-        name="phone"
+        name='phone'
         rules={[
           {
             required: true,
+            pattern: new RegExp(/\d+/g)
```

Comment:
there is another i try:

```js
<Form.Item
        label={translate('phone')}
        name='phone'
        rules={[
          {
            required: true,
          },
        ]}
        normalize={(value) => value.replace(/\D/g, '')}
      >
        <Input type='tel' />
      </Form.Item>
```
it Allows you to enter only numeric strings in the phone field.

Reply:
With each national, there is a different pattern. Can you handle it

Reply:
maybe we can add a util function like this：
```javascript
 const getPhoneValidationPattern = (lang) => {
    switch (lang) {
      case 'zh-TW':
        return { pattern: /^[0-9]{10}$/ };
      case 'en':
        return { pattern: /^[0-9]{9}$/ };

     // ...etc
      default:
        return { pattern: /\d+/g, };
    }
  };
```

@affkjuhyt What do you think